### DISCUSSION TITLE
현재 게임 레퍼런스 모음

### DISCUSSION THREAD

**Discussion by fkdl0048** [The-Day-In-1984/1984_Project#32] [Ideas]
## 전체적인 게임 디자인 레퍼런스

- https://sollerstech.itch.io/back-to-the-collis

## 미션 레퍼런스

- https://www.reddit.com/r/Unity2D/comments/11sq6rx/solving_some_puzzles_8_days_left_on_the/

**Comment by fkdl0048** [comment]
자유롭게 추가해주시면 감사하겠습니다..

**Comment by ccmuk58** [comment]
<h2>대화  레퍼런스</h2>

- https://steamcommunity.com/sharedfiles/filedetails/?id=1944643549

<h2>대화 구현 관련 레퍼런스</h2>

- [Excel to Unity1](https://velog.io/@gkswh4860/Unity-%EC%97%91%EC%85%80-%EB%8C%80%ED%99%94-%EB%82%B4%EC%9A%A9%EC%9D%84-%EB%8C%80%ED%99%94-%EC%9D%B4%EB%A6%84%EC%9C%BC%EB%A1%9C-%EB%AC%B6%EC%96%B4%EC%84%9C-%EA%B0%80%EC%A0%B8%EC%98%A4%EA%B8%B0)
- [Excel to Unity2](https://studybacksu.tistory.com/11)
- [NodeBasedDialogueSystem](https://github.com/merpheus-dev/NodeBasedDialogueSystem)
- [Dialogue](https://github.com/Siccity/Dialogue)
- [YarnSpinner](https://github.com/YarnSpinnerTool/YarnSpinner)

**Reply by fkdl0048** [reply]
정보 감사합니다아
